plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/Library.java

```diff
@@ -0,0 +1,30 @@
+package com.google.cloud.tools.eclipse.appengine.libraries;
+
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.Path;
+
+/**
+ * Represents a library that can be added to App Engine projects. E.g. AppEngine Endpoints library.
```

Comment:
Theoretically that could include something like Guava. Can this be more specific?